### PR TITLE
Victorherald

### PR DESCRIPTION
**Pull request by Victorherald**
did the markupp

### CODE REVIEW

File: class_six/GitHub-Mark.png
Comment (Sesughter01) [inline class_six/GitHub-Mark.png:1]: Good work but add this image to the assets folder in the root

File: class_six/Logo_of_Twitter.svg.png
Comment (Sesughter01) [inline class_six/Logo_of_Twitter.svg.png:1]: Good try but add this image to assets folder

File: class_six/asparagus-2169305_1280.jpg
Comment (Sesughter01) [inline class_six/asparagus-2169305_1280.jpg:1]: Add this image to the assets folder

File: class_six/football-1396740_1920.jpg
Comment (Sesughter01) [inline class_six/football-1396740_1920.jpg:1]: Add to the assets folder remove from here

File: class_six/index.html [hunk 1]
```diff
@@ -18,20 +18,22 @@ <h1>Grid Layout</h1>
         <div id="item1">Header</div>
```
Comment (Sesughter01) [inline class_six/index.html]: This element is supposed to hold your logo change the text copy to logo or better if you have a logo place here

File: class_six/pancakes-2291908_1280.jpg
Comment (Sesughter01) [inline class_six/pancakes-2291908_1280.jpg:1]: Add to the assets folder remove from here

File: class_six/salmon-518032_1280.jpg
Comment (Sesughter01) [inline class_six/salmon-518032_1280.jpg:1]: Add to the assets folder remove from here also use an image of equal size with the other images you have in this div. Good try.

File: class_six/vercel.png
Comment (Sesughter01) [inline class_six/vercel.png:1]: Add to the assets folder remove from here

**Review by Sesughter01 — CHANGES_REQUESTED** [review]
Please take a look at my comments and refactor (adjust) code accordingly

**Comment by Victorherald** [comment]
added it in asset folder

**Comment by Victorherald** [comment]
already corrected sir

**Comment by Victorherald** [comment]
corrected the folders

File: assets/GitHub-Mark.png
Comment (Sesughter01) [inline assets/GitHub-Mark.png:1]: corrected good work

File: assets/Logo_of_Twitter.svg.png
Comment (Sesughter01) [inline assets/Logo_of_Twitter.svg.png:1]: corrected good

File: assets/asparagus-2169305_1280.jpg
Comment (Sesughter01) [inline assets/asparagus-2169305_1280.jpg:1]: corrected

File: assets/football-1396740_1920.jpg
Comment (Sesughter01) [inline assets/football-1396740_1920.jpg:1]: ok

File: assets/images.jpg
Comment (Sesughter01) [inline assets/images.jpg:1]: I assume this is your logo . It is wrongly positioned it should be the header text was before correct that.
victorh_correctn

File: assets/pancakes-2291908_1280.jpg
Comment (Sesughter01) [inline assets/pancakes-2291908_1280.jpg:1]: ok good

File: assets/salmon-518032_1280.jpg
Comment (Sesughter01) [inline assets/salmon-518032_1280.jpg:1]: ok good

File: assets/vercel.png
Comment (Sesughter01) [inline assets/vercel.png:1]: ok

File: class_six/index.html
Comment (Sesughter01) [inline class_six/index.html:1]: your home, about, contact, login, and register links should be in the header not the side menu. You can put other things you like in the side menu
victorH2_correctns

**Review by Sesughter01 — CHANGES_REQUESTED** [review]
Please adjust your code based on the new corrections please go over the grid layout class notes again

**Comment by Victorherald** [comment]
adjusted the header logo to the left sir

File: class_six/GitHub-Mark.png
Comment (Sesughter01) [reply class_six/GitHub-Mark.png:1]: Why have you moved this image back to root . You were instructed to keep all your image assets inside the asset folder?

File: class_six/Logo_of_Twitter.svg.png
Comment (Sesughter01) [inline class_six/Logo_of_Twitter.svg.png:1]: move back to assets folder

File: class_six/asparagus-2169305_1280.jpg
Comment (Sesughter01) [inline class_six/asparagus-2169305_1280.jpg:1]: move back to assets folder

File: class_six/football-1396740_1920.jpg
Comment (Sesughter01) [inline class_six/football-1396740_1920.jpg:1]: move back to assets folder

Comment (Sesughter01) [inline class_six/football-1396740_1920.jpg:1]: move back to assets folder

File: class_six/pancakes-2291908_1280.jpg
Comment (Sesughter01) [inline class_six/pancakes-2291908_1280.jpg:1]: move back to assets

File: class_six/salmon-518032_1280.jpg
Comment (Sesughter01) [inline class_six/salmon-518032_1280.jpg:1]: move back to assets

File: class_six/vercel.png
Comment (Sesughter01) [inline class_six/vercel.png:1]: move back to assets folder

**Review by Sesughter01 — COMMENTED** [review]
Please follow the instructions you are given strictly . If you are not sure ask for clarification

**Comment by Victorherald** [comment]
SIR, here is my progress so far

**Comment by Victorherald** [comment]
created the javascrippt

**Comment by Victorherald** [comment]
done the markup

**Comment by Victorherald** [comment]
done the markup

**Comment by Sesughter01** [comment]
Good work on markup

**Review by Sesughter01 — APPROVED** [review]
Great work

**Pull request merged**